conda_smithy/cli.py

```diff
@@ -228,9 +228,11 @@ def __call__(self, args):
                         )
                         print("")
                     else:
+                        quote = "'" if not sys.platform.startswith('win') else '"'
                         print(
                             "You can commit the changes with:\n\n"
-                            "    git commit -m 'MNT: Re-rendered with conda-smithy %s'\n" % __version__
+                            "    git commit -m {q}MNT: Re-rendered with conda-smithy {version}{q}\n"
```

Comment:
Honestly I'd be ok just using `"` if that is better for Windows. Both should work fine on Unix as there is nothing fancy going on in the message.